modules/rosa-edit-objects.adoc

```diff
@@ -160,14 +160,20 @@ $ rosa edit machinepool --cluster=<cluster_name> | <cluster_id> <machinepool_ID>
 |--enable-autoscaling
 |Enable or disable autoscaling of compute nodes. To enable autoscaling, use this argument with the `--min-replicas` and `--max-replicas` arguments. To disable autoscaling, use `--enable-autoscaling=false` with the `--replicas` argument.
 
+|--labels
+|The labels (string) for the machine pool. The format must be a comma-delimited list of key=value pairs. This list overwrites any modifications made to node labels on an ongoing basis.
+
 |--max-replicas
 |Specifies the maximum number of compute nodes when enabling autoscaling.
 
 |--min-replicas
 |Specifies the minimum number of compute nodes when enabling autoscaling.
 
 |--replicas
-|The number (integer) of machines for this machine pool.
+|Required when autoscaling is not configured. The number (integer) of machines for this machine pool.
+
+|--taints
```

Comment:
The taints and label edit will only work for the new created nodes of the machine pool. Should we clarify it? For example, there is a machinepool with 2 nodes, and then user edit the taints, the new taints won't affect the existing nodes. If user scale up the nodes number to 3. Then the new created node will have the edited taints.

Reply:
Added the clarification to taints and labels. Thanks.